### ISSUE TITLE
backend empty when updating the page builder

### ISSUE THREAD

**Issue by zanicom** [ThemeFuse/Unyson#3649]
The page builder is working well on my site (Version 2.6.16) but there is a vulnerability.  So when I try to update the plugin, the page builder becomes empty on the backend but OK on the frontend. (See screenshots)

screenshot 1- Working Plugin version
screnshot 2- Backend page before updating the plugin
Screenshot 3 - PLugins updated
Screenshot 4 - Backend empty
Screenshot 5 - Frontend still OK

![05](https://user-images.githubusercontent.com/45213172/48799296-bf35dc00-ecd4-11e8-8c09-80e34eaa69f4.png)
![04](https://user-images.githubusercontent.com/45213172/48799297-bf35dc00-ecd4-11e8-8f14-b24901a5dc70.png)
![03](https://user-images.githubusercontent.com/45213172/48799299-bf35dc00-ecd4-11e8-9aa1-6a1973d9c6f0.png)
![01](https://user-images.githubusercontent.com/45213172/48799300-bf35dc00-ecd4-11e8-8da5-76fcd8c8da40.png)
![02](https://user-images.githubusercontent.com/45213172/48799301-bfce7280-ecd4-11e8-9045-4b9616be9168.png)

Can you please help me?  Thanks!

**Comment by danyj** [comment]
that is not  vulnerability   it is called a bug , vulnerability means something completely different and it is dangerous.

**Comment by KirillFE** [comment]
Greetings!
Unfortunately, I have same issue `Version 4.9.8 | Unyson 2.7.20`

**Comment by zanicom** [comment]
@danyj:  Yeah...sorry...  I meant that there is a vulnerability on the version of the plugin so that is why I needed to update it...  And after updating it, there was that bug!!
